### ISSUE TITLE
Incorrect maximum for ses_event_destination

### ISSUE THREAD

**Issue by david-wells-1** [terraform-providers/terraform-provider-aws#6689]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.11.10

### Affected Resource(s)

* resource_aws_ses_event_destination

### Terraform Configuration Files

```hcl
resource "aws_ses_configuration_set" "dev" {
  name = "hcom"

  lifecycle {
    create_before_destroy = "true"
  }
}

resource "aws_ses_event_destination" "cloudwatch-all" {
  name                   = "dev-all"
  configuration_set_name = "${aws_ses_configuration_set.dev.name}"
  enabled                = true
  matching_types         = ["bounce", "complaint", "reject", "send", "delivery"]

  cloudwatch_destination = {
    default_value  = "${aws_ses_configuration_set.dev.name}"
    dimension_name = "ses:configuration-set"
    value_source   = "messageTag"
  }

  cloudwatch_destination = {
    default_value  = "default"
    dimension_name = "ses:caller-identity"
    value_source   = "messageTag"
  }

  cloudwatch_destination = {
    default_value  = "default"
    dimension_name = "ses:from-domain"
    value_source   = "messageTag"
  }

  cloudwatch_destination = {
    default_value  = "default"
    dimension_name = "ses:source-ip"
    value_source   = "messageTag"
  }

  cloudwatch_destination = {
    default_value  = "default"
    dimension_name = "ses:outgoing-ip"
    value_source   = "messageTag"
  }

  cloudwatch_destination = {
    default_value  = "default"
    dimension_name = "to"
    value_source   = "emailHeader"
  }
}
```

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by bflad** [comment]
Support for multiple `cloudwatch_destination` configuration blocks has been merged and will release with version 1.51.0 of the AWS provider, in the next day or two. 👍

**Comment by bflad** [comment]
This has been released in [version 1.51.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1510-december-05-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
